python/deltalake/schema.py

```diff
@@ -1,7 +1,40 @@
-from typing import Union
+from typing import Tuple, Union
+
+import pandas as pd
+import pyarrow as pa
```

Comment:
I *think* this is the last thing: We'll want to keep Pandas an optional dependency:

```suggestion
from typing import Tuple, Union, TYPE_CHECKING

import pyarrow as pa

if TYPE_CHECKING:
    import pandas as pd
```